WordPress/twentytwentyone
# Remove the link on published date

Fixes https://github.com/WordPress/twentytwentyone/issues/481

## Summary


Unlinks the publish date

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Go to the archive
1. See that the published date aren't linked anymore


## Quality assurance
